README.md

```diff
@@ -38,7 +38,9 @@ Mock -> WIP -> alpha -> beta -> production
 
 ## Documentation
 
-1. [Database interacitons](docs/db.md)
+1. Database
```

Comment:
Not relate to this line. 
But we should update to "CometBFT blocks to update the Native Light Client in the Ika network"
Same for "Bitcoin blocks..."